packages/@n8n/task-runner/src/config/base-runner-config.ts

```diff
@@ -23,8 +23,13 @@ export class BaseRunnerConfig {
 	@Env('N8N_RUNNERS_MAX_PAYLOAD')
 	maxPayloadSize: number = 1024 * 1024 * 1024;
 
+	/**
+	 * How many concurrent tasks can a runner execute at a time
+	 *
+	 * Kept high for backwards compatibility - n8n v2 will reduce this to `5`
+	 */
 	@Env('N8N_RUNNERS_MAX_CONCURRENCY')
-	maxConcurrency: number = 5;
+	maxConcurrency: number = 10;
```

Comment:
what's the reason to define these a second time? could we delete these, and use `TaskRunnersConfig` from the config package instead?